adaptive_components/test/adaptive_components_test.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter_test/flutter_test.dart';
```

Comment:
Delete this file if there not going to be any tests, otherwise add some tests

Reply:
DOne